.github/workflows/universalnrp-test.yml

```diff
@@ -27,8 +28,15 @@ jobs:
             { os: ubuntu, version: 22.04, package-type: DEB, tag: ''},
           ]
         arch: [amd64]
-        install-osconfig: [false]
-        policy-package: [LinuxSshServerSecurityBaselinePolicy.zip]
+        install-osconfig: [true, false]
+        policy-package: [LinuxSshServerSecurityBaselinePolicy.zip, LinuxSecurityBaselinePolicy.zip]
+        exclude:
+          # LinuxSshServerSecurityBaselinePolicy.zip must run without OSConfig
+          - policy-package: LinuxSshServerSecurityBaselinePolicy.zip
+            install-osconfig: true
+          # LinuxSecurityBaselinePolicy.zip must run with OSConfig
```

Comment:
Actually, both of these tests now should run without OSConfig, all by themselves. The Universal NRP defaults to the 'OSConfig for MC' aka itself for both SSH Posture Control and ASB v2 policies.